package.json

```diff
@@ -55,17 +55,17 @@
     "@types/js-combinatorics": "0.5.29",
     "@types/jspdf": "^1.1.31",
     "@types/jstree": "^3.3.35",
-    "@types/lodash": "^4.14.104",
+    "@types/lodash": "4.14.104",
```

Comment:
more recent versions of lodash typing have some weird issue that completely messes up a lot of type inference